cli.js

```diff
@@ -9,9 +9,10 @@ const args = require('minimist')(process.argv.slice(2), {
     missing: false,
     extra: false,
     dev: true,
-    'default-entries': true
+    'default-entries': true,
+    quiet: false,
```

Comment:
There's a trailing comma here, which the https://standardjs.com/ coding style guide we use don't allow, hence the failed tests. Could you remove it and push an update here? Then the tests should turn green and we're good to go 🙂 🚀 🎸